R/data_copy.R

```diff
@@ -130,7 +130,7 @@ spark_serialize_csv_scala <- function(sc, df, columns, repartition) {
     "sparklyr.Utils",
     "createDataFrameFromCsv",
     spark_context(sc),
-    tempfile,
+    paste("file://", tempfile, sep = ""),
```

Comment:
I agree. Actually, if the issue is being reported with `master="local"`, we might rather want to replace [csv_file ](https://github.com/rstudio/sparklyr/blob/master/R/data_copy.R#L154) with `csv_file_scala`, since currently `csv_file` makes use of of `spark_read_csv` which is probably trying to read from HDFS in some corner cases when our expectation was to read from local. Long term, we want to move away from these custom serializers and use Apache Arrow / Feather, that is going to be a great day. Going to close this PR and follow up once we know what environment is being affected, I think we need to know that at the very minimum.